lib/lita/handlers/locker.rb

```diff
@@ -42,6 +42,13 @@ class Locker < Handler
       )
 
       route(
+        /^give\s#{LABEL_REGEX}\sto\s#{USER_REGEX}#{COMMENT_REGEX}$/,
```

Comment:
Since this isn't likely to be super common, I'd say it's better to put this in the locker namespace? `^locker\sgive\s...`?